package.nls.json

```diff
@@ -93,6 +94,7 @@
     "AWS.message.credentials.invalid.help": "Get Help...",
     "AWS.message.credentials.invalid.logs": "View Logs...",
     "AWS.message.enterProfileName": "Enter the name of the credential profile to use",
+    "AWS.message.info.submitFeedback.success": "Thanks for the feedback!",
```

Comment:
Nit: Not sure what kind of a voice we want to use for the toolkit. I would probably opt for "Thank you for the feedback!"

Reply:
Adopted the phrasing from the jb.

https://github.com/aws/aws-toolkit-jetbrains/blob/c203519a8da328e12b46777b574b1581920e8225/resources/resources/software/aws/toolkits/resources/localized_messages.properties#L563

Lets leave this for now for consistency and discuss a phrasing overhaul we can do across the board in the future.